ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dot1x.j2

```diff
@@ -79,5 +97,30 @@ dot1x
    radius av-pair framed-mtu {{ dot1x.radius_av_pair.framed_mtu }}
 {%             endif %}
 {%         endif %}
+{%         if dot1x.supplicant.disconnect_cached_results_timeout is arista.avd.defined %}
+   supplicant disconnect cached-results timeout {{ dot1x.supplicant.disconnect_cached_results_timeout }} seconds
+{%         endif %}
+{%         if dot1x.captive_portal.enabled is arista.avd.defined(true) %}
+   captive-portal
+{%         endif %}
+{%         if dot1x.captive_portal.url is arista.avd.defined or dot1x.captive_portal.ssl_profile is arista.avd.defined %}
```

Comment:
i think we need to use elif here

Reply:
We can give both as well
`captive-portal url http:shivani:40 ssl profile pf1`